src/devices/oregon_scientific.c

```diff
@@ -273,16 +273,16 @@ static int oregon_scientific_v2_1_decode(r_device *decoder, bitbuffer_t *bitbuff
     else if (sensor_id == ID_BHTR968) {
         if (validate_os_v2_message(decoder, msg, 92, msg_bits, 19) != 0)
             return 0;
-        unsigned int comfort = msg[7] >> 4;
-        char *comfort_str = "Normal";
-        if (comfort == 4) comfort_str = "Comfortable";
-        else if (comfort == 8) comfort_str = "Dry";
-        else if (comfort == 0xc) comfort_str = "Humid";
-        unsigned int forecast = msg[9] >> 4;
-        char *forecast_str = "Cloudy";
-        if (forecast == 3) forecast_str = "Rainy";
-        else if (forecast == 6) forecast_str = "Partly Cloudy";
-        else if (forecast == 0xc) forecast_str = "Sunny";
+        //unsigned int comfort = msg[7] >> 4;
+        //char *comfort_str = "Normal";
+        //if (comfort == 4) comfort_str = "Comfortable";
+        //else if (comfort == 8) comfort_str = "Dry";
+        //else if (comfort == 0xc) comfort_str = "Humid";
+        //unsigned int forecast = msg[9] >> 4;
+        //char *forecast_str = "Cloudy";
+        //if (forecast == 3) forecast_str = "Rainy";
+        //else if (forecast == 6) forecast_str = "Partly Cloudy";
+        //else if (forecast == 0xc) forecast_str = "Sunny";
```

Comment:
Unsure. This looks like prototype code that some one never finished.

Reply:
Yeah. Figured it might be useful for later reference or as a starting point if anyone wants to expose these fields. Maybe we could add this info as comment in the section above the code if anyone wants to work on this at a later moment.